docs/pages/agents/introduction.mdx

```diff
@@ -67,11 +67,41 @@ agents](deploy-agents-terraform.mdx).
 
 ## Joining agents
 
+### Initally joining a cluster
+
 Teleport agents need to establish trust with the Teleport Auth Service in order
-to authorize users to access your infrastructure. There are several ways to join
-an agent to your Teleport cluster, making it possible to automate the join
-process for your environment. Read about the available join methods in our [Join
-Services to your Cluster](./join-services-to-your-cluster.mdx) guides.
+to join a cluster. There are several ways to join an agent to your Teleport
+cluster, making it possible to automate the join process for your environment.
+Read about the available join methods in our [Join Services to your
+Cluster](./join-services-to-your-cluster.mdx) guides.
+
+When a Teleport process first runs on an agent, it checks its configuration file
```

Comment:
does the process run on the agent, or does the process serving as an agent run on the host?

Reply:
I would say the latter.